server/src/app.ts

```diff
@@ -1,204 +1,253 @@
 import express from "express";
 import cors from "cors";
-import bodyParser, { text } from "body-parser";
+import bodyParser from "body-parser";
 import "./db";
+import User from "./models/user.model";
+import Memory from "./models/memory.model";
+import Person from "./models/person.model";
 
 const app = express();
 const port = 3000;
 
-import User from "./models/user.model";
-import Memory from "./models/memory.model";
-import Person from "./models/person.model";
 
 app.use(
-  cors({
-    origin: "*",
-  })
+    cors({
+        origin: "*",
+    })
 );
-
 app.use(bodyParser.json());
 
 //@ts-ignore
 app.listen(port, (err) => {
-  console.log("Listening and we know of cors:", cors);
-  if (err) {
-    return console.error(err);
-  }
-  return console.log(`server is listening on ${port}`);
+    console.log("Listening and we know of cors:", cors);
+    if (err) {
+        return console.error(err);
+    }
+
+    return console.log(`server is listening on ${port}`);
 });
 
+
 const register = app.post("/api/register", async (req, res, next) => {
-  const userInput = req.body;
+    const userInput = req.body;
 
-  async function createUser(userModel: any, userInput: any) {
-    const newUser: any = await userModel.create({
-      username: userInput.username,
-      mail: userInput.mail,
-    });
 
-    return newUser;
-  }
+    async function createUser(userModel: any, userInput: any) {
+        const newUser: any = await userModel.create({
+            username: userInput.username,
+            mail: userInput.mail,
+        });
 
-  const newUser = await createUser(User, userInput)
+        return newUser;
+    }
```

Comment:
Do validation on `userInput`: make sure it has `email` and `password`, and that they are strings.